File: GarageGo-BE/Models/LienHe.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace GarageGo_BE.Models
{
    [Table("LIENHE")]
    public class LienHe
    {
        [Key]
        public int MaLienHe { get; set; }

        [Required]
        [StringLength(100)]
        public string HoTen { get; set; } = string.Empty;

        [Required]
        [StringLength(100)]
        public string Email { get; set; } = string.Empty;

        [Required]
        [StringLength(20)]
        public string SoDienThoai { get; set; } = string.Empty;

        [StringLength(100)]
        public string? ChuDe { get; set; }

        [Required]
        public string NoiDung { get; set; } = string.Empty;

        public DateTime NgayGui { get; set; } = DateTime.Now;

        public bool DaXuLy { get; set; } = false;

        [StringLength(50)]
        public string? Username { get; set; } // Username của user đã liên hệ
    }
}

File: GarageGo-BEJava/src/main/java/com/garagego/model/LienHe.java
package com.garagego.model;

import jakarta.persistence.*;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "LIENHE")
public class LienHe {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "MaLienHe")
    private Integer maLienHe;

    @Column(name = "HoTen", nullable = false, length = 100)
    private String hoTen;

    @Column(name = "Email", nullable = false, length = 100)
    private String email;

    @Column(name = "SoDienThoai", length = 20)
    private String soDienThoai;

    @Column(name = "ChuDe", length = 200)
    private String chuDe;

    @Column(name = "NoiDung", length = 2000)
    private String noiDung;

    @Column(name = "NgayGui")
    private LocalDateTime ngayGui = LocalDateTime.now();

    @Column(name = "DaXuLy")
    private Boolean daXuLy = false;

    @Column(name = "Username", length = 50)
    private String username;
}
